src/layouts/index.js

```diff
@@ -71,14 +71,18 @@ const HeaderDesktop = ({ language, hasLanguage, setLanguage }) => {
           <strong>eBPF Summit</strong>
           <Link className="link" to="/summit-2021">2021</Link>
           <Link className="link" to="/summit-2020">2020</Link>
+          <strong>eBPF Day (KubeCon)</strong>
+          <a target="_blank" className="link" href="https://events.linuxfoundation.org/cloud-native-ebpf-day-europe/program/cfp/">2022 CFP</a>
+          <a target="_blank" className="link" href="https://events.linuxfoundation.org/cloud-native-ebpf-day-north-america/program/schedule/">2021</a>
           <strong>eBPF Track (LPC)</strong>
           <a target="_blank" className="link" href="https://linuxplumbersconf.org/event/11/sessions/120/#all">2021</a>
           <a target="_blank" className="link" href="https://linuxplumbersconf.org/event/7/sessions/91/#all">2020</a>
           <a target="_blank" className="link" href="https://linuxplumbersconf.org/event/4/sessions/62/#20190911">2019</a>
           <a target="_blank" className="link" href="http://vger.kernel.org/lpc-bpf2018.html">2018</a>
           <a target="_blank" className="link" href="https://blog.linuxplumbersconf.org/2017/ocw/events/LPC2017/schedule.html#day_2017_09_15">2017</a>
           <strong>bpfconf (LSF/MM/BPF)</strong>
-          <a target="_blank" className="link" href="https://lore.kernel.org/bpf/60997079.1c69fb81.77f3f.a045@mx.google.com/">2021 CFP</a>
+          <a target="_blank" className="link" href="https://lore.kernel.org/bpf/YfGnDRM%2FPe4jzbSr@localhost.localdomain/">2022 CFP</a>
+          <a target="_blank" className="link" href="https://lore.kernel.org/bpf/60997079.1c69fb81.77f3f.a045@mx.google.com/"><s>2021</s></a>
```

Comment:
Why the `<s>...</s>` in this line?

Reply:
The 2021 eventually got cancelled and is postponed to 2022 given it's an onsite-only event and corona situation didn't allow for it.